Frontend/ui-common/src/Components/RideCompletedCard/Controller.purs

```diff
@@ -96,10 +117,26 @@ config = {
     imageWidth : V 0, 
     imageHeight : V 0
   },
+  driverUpiQrCard : {
+    text : "Get direct payment to UPI ID",
```

Comment:
Is this data used ?
If yes then can we add translations
If no then It can be removed